website/integrations/services/jenkins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Jenkins
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Jenkins
+
+> The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.
```

Comment:
```suggestion
> Jenkins is open source automation server, with hundreds of plugins to support building, deploying, and automating any project.
```

Reply:
this is taken 1:1 from their webpage so I'll keep it as is